README.md

```diff
@@ -94,11 +94,11 @@ I would like to acknowledge Microverse community for thier support.
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
+ 
+## 📝 License <a name="license"></a> 
 
-## 📝 License <a name="license"></a>  
-
-This project is [MIT](./LICENSE) licensed.
+This project is [MIT](./MIT.md) licensed.
 
```

Comment:
- [ ] As requested by the previous reviewer https://github.com/Enoisong/Mobile-First-Portfolio/pull/1#discussion_r1109397445 please make sure the MIT license link in your Readme is not broken as currently it is giving a 404 error, please add a LICENSE file to the root folder and fix this.